build/root/Makefile

```diff
@@ -279,6 +279,36 @@ test-e2e-node: ginkgo generated_files
 	hack/make-rules/test-e2e-node.sh
 endif
 
+define TEST_E2E_KUBEADM_HELP_INFO
+# Build and run kubeadm end-to-end tests.
+#
+# Args:
+#  FOCUS: Regexp that matches the tests to be run.  Defaults to "".
+#  SKIP: Regexp that matches the tests that needs to be skipped.  Defaults
+#    to "".
+#  RUN_UNTIL_FAILURE: If true, pass --untilItFails to ginkgo so tests are run
+#    repeatedly until they fail. Defaults to false.
+#  ARTIFACTS: Local directory to save test artifacts into. Defaults to "/tmp/_artifacts".
+#  PARALLELISM: The number of gingko nodes to run.  If empty ginkgo default 
+#    parallelism (cores - 1) is used
+#  BUILD: Build kubeadm end-to-end tests. Defaults to true.
```

Comment:
side note: I'm amused that we're adding a make rule which calls `go run` which then calls `bazel build`.